wikilegis/static/javascript/content/modules/amendmentDiff.js

```diff
@@ -3,32 +3,32 @@ import $ from 'jquery';
 const diff = require('diff');
 
 function amendmentDiffModule() {
-    function updateDiff(inputEl) {
-        const containerEL = $(inputEl).closest('[data-segment-content]')[0]
-        const segmentContent = containerEL.dataset.segmentContent;
-        const diffEl = buildMarkup(diff.diffWordsWithSpace(segmentContent, inputEl.value));
+  function buildMarkup(oldText, newText) {
+    const diffElement = document.createDocumentFragment();
+    diff.diffWordsWithSpace(oldText, newText).forEach((part) => {
+      let className = '';
+      if (part.added) className = 'diff-added';
+      else if (part.removed) className = 'diff-removed';
 
-        const diffWrapperEl = containerEL.querySelector('[data-diff-wrapper]');
-        diffWrapperEl.innerHTML = '';
-        diffWrapperEl.appendChild(diffEl);
-    }
+      const span = document.createElement('span');
+      $(span).addClass(className);
```

Comment:
Try using vanilla JS. Specially if you created a polyfill specifically for this right here lol